deployments/apiv2/services/users/src/main.py

```diff
@@ -408,7 +408,7 @@ async def get_all_users(
     customer_id = uuid.UUID(hex=token["userid"])
 
     query = (
-        "SELECT id, name, email, created_at, last_login, suspended FROM users "
+        "SELECT id, LOWER(name), email, created_at, last_login, suspended FROM users "
```

Comment:
Is the `LOWER` for extra safety measures? They should all be lowercase in the DB correct?

Reply:
I guess it is redundant. I will remove it